x-pack/legacy/plugins/siem/public/components/help_menu/index.tsx

```diff
@@ -24,10 +24,24 @@ export const HelpMenu = React.memo(() => {
           href: docLinks.links.siem.guide,
           iconType: 'documents',
           linkType: 'custom',
+          target: '_blank',
+          rel: 'noopener',
+        },
+        {
+          content: i18n.translate('xpack.siem.chrome.helpMenu.documentation.ecs', {
+            defaultMessage: 'ECS documentation',
+          }),
+          href: `${docLinks.ELASTIC_WEBSITE_URL}guide/en/ecs/${docLinks.DOC_LINK_VERSION}/index.html`,
```

Comment:
What's the value of `DOC_LINK_VERSION` here? If this is the stack version, this will 404. If you're maintaining a mapping between SIEM versions and ECS versions, I still think this is risky.

I think the simplest approach would likely be best, here, and simply link to "current".

Reply:
That's right! Forgot about that 🤦‍♂ Thanks @webmat! Will update to just be `current`. 

As an aside, is there anywhere we maintain a mapping to what the current ECS version is at the time of Kibana/SIEM releases?

Reply:
> is there anywhere we maintain a mapping to what the current ECS version is at the time of Kibana/SIEM releases?

Short answer: not really.

Long answer :-)

I don't necessarily think it's a good idea to even try to track a mapping between different versions. Here's why:

Right now Beats declares globally an ECS version for all sources at every product version, and puts that in all event's `ecs.version` field. This happens in libbeat, whether or not a given source was adjusted to recent additions in ECS.

From the beginning I've tried to advocate for setting the appropriate ECS version per source, and adjust it only when the source is modified to adjust to new developments in ECS. It was deemed too much maintenance initially, but the current "global ECS version" has been causing confusion. So the discussion is on the table again: https://github.com/elastic/beats/issues/16089.

If we go the direction of declaring a more precise ECS version per source, then mapping between a product version and an ECS version becomes moot.

All of this complexity would be unnecessary if we upgraded all sources in lockstep with new developments in ECS. But that hasn't been happening so far.

---

For similar reasons to Beats, I'm not sure I'd try to maintain a mapping between Elastic SIEM and ECS.

Perhaps SIEM upgrading all visualizations and queries in lockstep with ECS is easier than the Beats situation, I don't know. If so, then we could maintain that and publish it somehow.

I actually don't know if there's already such a mapping anywhere in SIEM.

Reply:
Thanks for the details + history @webmat -- this is helpful 🙂 

Upgrading our visualizations/queries in lockstep with ECS sounds like the right path forward, and feels in line with what we did during initial development. There were often ECS fields that visualizations were using that beats hadn't been updated to utilize yet (`asn` comes to mind), and so that field would just be empty till the beat supported it, but the visualization would at least be in sync with ECS and ready to ship.

As for any such version mapping within SIEM itself, I don't believe there is. We do however maintain a few ECS field mappings like [here](https://github.com/elastic/kibana/blob/7ac8e4d9ccc2a9e064d8a5621cd62453d1006164/x-pack/legacy/plugins/siem/server/utils/beat_schema/8.0.0/ecs.ts) and [here](https://github.com/elastic/kibana/blob/a157513cfa1512c732001d040259d28c435f371d/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/index/ecs_mapping.json) though, which is a process that could probably use a revisit to ensure we're staying in sync with each ECS release as they happen.